src/repr/src/row/collection.proto

```diff
@@ -14,6 +14,7 @@ package mz_repr.row.collection;
 message ProtoRowCollection {
   bytes encoded = 1;
   repeated ProtoEncodedRowMetadata metadata = 2;
+  repeated uint64 fingers = 3;
```

Comment:
"fingers" is an interesting term for this, what do you think about calling this field "runs"? It matches terminology from Persist for a similar concept

Reply:
Changing to `runs`.